HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -10,6 +10,7 @@ SERVER=triton_server_instance
 RETRIES=3
 REPOS=()
 MODELS=()
+ENSEMBLEMODELS=()
```

Comment:
this variable name is confusing, because this array actually holds the submodel names, not the ensemble models themselves.